### PR TITLE
Project Metrics: Add additional level to metric scores

### PR DESCRIPTION
**Pull request by rcebulko**
Syncs with levels in design doc.

### CODE REVIEW

File: project-metrics/metrics_service/metrics/travis_greenness.py [hunk 1]
```diff
@@ -14,10 +14,12 @@ class TravisGreennessMetric(base.PercentageMetric):
 
   def _score_value(self, percentage: float) -> models.MetricScore:
     if percentage < 0.6:
+      return models.MetricScore.CRITICAL
+    elif percentage < 0.74:
```
Comment (danielrozenberg) [inline project-metrics/metrics_service/metrics/travis_greenness.py]: I thought that was a finger slip at first 🙃

File: project-metrics/metrics_service/metrics/travis_greenness_test.py [hunk 2]
```diff
@@ -22,7 +22,7 @@ def testRecompute(self):
     session = self.Session()
     session.add_all([
         models.Build(state=models.TravisState.PASSED, **common_values),
-        models.Build(state=models.TravisState.CANCELLED, **common_values),
+        models.Build(state=models.TravisState.CANCELED, **common_values),
```
Comment (danielrozenberg) [inline project-metrics/metrics_service/metrics/travis_greenness_test.py]: lol

File: project-metrics/metrics_service/metrics/travis_greenness.py (hunk 1 above)
Comment (rcebulko) [reply project-metrics/metrics_service/metrics/travis_greenness.py]: Yeah weird threshold but w/e

File: project-metrics/metrics_service/metrics/travis_greenness_test.py (hunk 2 above)
Comment (rcebulko) [reply project-metrics/metrics_service/metrics/travis_greenness_test.py]: This *was* a finger slip lol

**Pull request merged**